Mathlib/LinearAlgebra/Matrix/PosDef.lean

```diff
@@ -217,4 +361,25 @@ def InnerProductSpace.ofMatrix {M : Matrix n n 𝕜} (hM : M.PosDef) :
   InnerProductSpace.ofCore _
 #align matrix.inner_product_space.of_matrix Matrix.InnerProductSpace.ofMatrix
 
+open Lean PrettyPrinter.Delaborator SubExpr in
+/-- Custom elaborator to produce output like `(_ : PosSemidef A).sqrt` in the goal view. -/
+@[delab app.Matrix.PosSemidef.sqrt]
+def delabSqrt : Delab :=
+  whenPPOption getPPNotation <|
+  whenNotPPOption getPPAnalysisSkip <|
+  withOptionAtCurrPos `pp.analysis.skip true do
+    let e ← getExpr
+    guard <| e.isAppOfArity ``Matrix.PosSemidef.sqrt 7
+    let optionsPerPos ← withNaryArg 6 do
+      return (← read).optionsPerPos.setBool (← getPos) `pp.proofs.withType true
+    withTheReader Context ({· with optionsPerPos}) delab
```

Comment:
It would be best to put this immediately below `def sqrt` so that the printing takes effect when navigating the proofs in the file.

Reply:
Done